### DISCUSSION TITLE
Is it possible to set a start date for a user?

### DISCUSSION THREAD

**Discussion by spenol** [kimai/kimai#5051] [General] [answered]
Hi,

Is it possible to set a start date for a user, so that when expected working hours are set, days before they joined are not shown up as red in the Employment contract screen? This would also allow the overall expected hours not to include times that they weren't working.

Thanks,

Oliver.

**Comment by kevinpapst** [comment, marked as answer]
Yes, with the https://www.kimai.org/store/controlling.html plugin

**Reply by spenol** [reply]
Thank you @kevinpapst
